daemon/api.go

```diff
@@ -211,7 +211,7 @@ func loginUser(c *Command, r *http.Request) Response {
 			Type: ResponseTypeError,
 			Result: &errorResult{
 				Kind:    errorKindTwoFactorRequired,
-				Message: "two factor authentication required",
+				Message: store.ErrAuthenticationNeeds2fa.Error(),
```

Comment:
That's not very effective unfortunately. The client and server may be running different versions of Snappy, with different messages or even translations eventually. That's why we have the Kind right above.